### ISSUE TITLE
Request to stop camera from zooming in when locking on

### ISSUE THREAD

**Issue by CeaseNY716** [Lyall/FFXVIFix#56]
Seems like this would fall in the scope of this mod

**Comment by CeaseNY716** [comment]
Yeah whatever this is, it does it in training hall as well. I thought it may be because of fov or having my camera centered but it still does it on default settings. Was a request by somebody on discord, which made me notice it more

https://github.com/user-attachments/assets/5562cae3-3992-46fd-9767-6e632b2e1fb9

**Comment by Geuschbenst11** [comment]
https://github.com/user-attachments/assets/830cdb30-2612-48b4-b8c5-64650900581e

I'm not sure if because I'm in the open area but it does zoom.
